### ISSUE TITLE
[FEATURE REQUEST] Enable Electrum server SSL connection with automatic get SSL certificate + onion Tor address connection

### ISSUE THREAD

**Issue by twofaktor** [nunchuk-io/nunchuk-android#39]
Hello, same as other wallets, Electrum, Sparrow, etc, this is a request to add a new feature to Nunchuk Android on Network settings

After testing the Electrum server connection using the SSL port to a secure connection it doesn't work:

![Screenshot_20231124_195932_io nunchuk android_edit_18711835955476](https://github.com/nunchuk-io/nunchuk-android/assets/89636253/92a4b5ca-c464-49b0-a8d5-4f7614779272)![Screenshot_20231124_200237_io nunchuk android](https://github.com/nunchuk-io/nunchuk-android/assets/89636253/e45de95a-c02d-4057-a4f2-29157da48c75)![Screenshot_20231124_200537_io nunchuk android_edit_19044033908029](https://github.com/nunchuk-io/nunchuk-android/assets/89636253/58e80afa-1e17-4fa1-b488-8885d89b82d9)

I suggest the following features:

- Enable compatibility with a secure connection to a personal Electrum Server using SSL
- Obtain the SSL certificate automatically without the need to transfer it manually from the server
- Enable availability to connect to Electrum Servers using its onion address Tor, integrating a Tor instance on the Nunchuk app or in conjunction with Orbot app (SSL not needed)

Thanks!

**Comment by PeteClubSeven** [comment]
Yeah I agree it's a lot clearer that way, I hope they implement more options 👍

**Comment by twofaktor** [comment]
This issue seems to have been abandoned, with no response from the team after so much time. If you ever consider integrating it, please go ahead and revive it. Closing...
